### PR TITLE
docs: logo optimization and adding alt

### PR DESCRIPTION
**Pull request by Amyssyko**
The JPG image has been replaced with an AVIF version, which offers better compression and faster loading times, AVIF (which is supported by 92.64% of browsers worldwide according to [caniuse](https://caniuse.com/avif)). This optimizes page performance without compromising compatibility or image quality.

### CODE REVIEW

**Review by dai-shi — CHANGES_REQUESTED** [review]
Just leave the image file unchanged for now. ALT seems good.

**Comment by Amyssyko** [comment]
I understand the request to keep the current image. To implement the alt text and also improve performance with AVIF, we could use the following structure:
```js
<div class="flex justify-center mb-4">
  <picture>
    <source srcset="../bear.avif" type="image/avif" />
    <img src="../bear.jpg" alt="Logo Zustand" />
  </picture>
</div>
```
Keep the image used as a fallback and optimize it

**Comment by dai-shi** [comment]
I understand your suggestion, thanks. But, let's just leave it.

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ nav: 0
 ---
 
 <div class="flex justify-center mb-4">
-  <img src="../bear.jpg" />
+    <img src="../bear.jpg" alt="Logo Zustand" />
```
Comment (dai-shi) [inline docs/getting-started/introduction.md:8]:
```suggestion
  <img src="../bear.jpg" alt="Logo Zustand" />
```

**Review by dai-shi — APPROVED** [review]
thx!

**Pull request merged**